src/gh.ts

```diff
@@ -371,6 +371,49 @@ const completionSpec: Fig.Spec = {
         },
       ],
     },
+    {
+      name: "browse",
+      description: "Open the repository in the browser",
```

Comment:
The browse command also accepts arguments, we may provide generators for this but I think we can tackle that in another PR
```suggestion
      description: "Open the repository in the browser",
      args: {
        name: "pr | issue | path[:line]"
      },
```

Reply:
I added the existing generator `ghGenerators.listPR` to this argument. Not sure how you feel about using a generator that only satisfies part of the argument.